app/src/main/java/io/neurolab/NeuroSettings.java

```diff
@@ -1,13 +1,29 @@
 package io.neurolab;
 
 
+import android.annotation.TargetApi;
+import android.os.Build;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+@TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
Why?

Reply:
@iamareebjamal To use `ConcurrentLinkedDeque` . @suresh-gandhi An error is shown while using it, Though even without adding `@TargetApi `, it runs on devices having below APIs as well. A similar reference can be found here - https://stackoverflow.com/questions/32090527/concurrentlinkeddeque-in-api-below-21